Dockerfile.init

```diff
@@ -0,0 +1,29 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:latest as init-builder
+
+RUN curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+RUN chmod +x /bin/gimme
+RUN yum upgrade -y && yum install -y tar gzip git make gcc
```

Comment:
do we need all of these packages?

Reply:
After verification, `tar gzip git` are required. Already remove `make gcc`. Thanks for this reminder~